LICENSE.md

```diff
@@ -1,16 +0,0 @@
-## License
```

Comment:
Seems like we should move all this to our README since it does seem relevant&hellip;

See bottom section: https://github.com/OpenINF/.github/blob/main/README.md#contributing